File: programme/years-to-days/years-to-days.cs
// Write a C# programme to convert years to days
using System;

public class YearsToDays
{
    static void Main(string[] args)
    {
        int YearsToDays(int years)
        {
            return years * 365;
        }

        // Input number of years
        Console.Write("Years: ");
        int years = int.Parse(Console.ReadLine());

        // Days output
        Console.WriteLine("Days: " + YearsToDays(years));
    }
}

File: programme/years-to-days/years-to-days.js
// Write a programme to convert years to days

"use strict";

// perform following command to run programme
// node x-power-y.js

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question('Input : ', years => {
	const fyears = parseFloat(years);
	if (isNaN(fyears)) {
		console.error('Some error occured');
	} else {
		console.log(`Output : ${fyears * 365}`);
	}
	rl.close();
});
